Packs/SafeNet_Trusted_Access/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,27 @@
+#### Integrations
+##### SafeNet Trusted Access
+Added 5 commands:
+- ***sta-get-application-list***
+- ***sta-get-application-info***
+- ***sta-get-user-applications***
+- ***sta-get-user-sessions***
+- ***sta-delete-user-sessions***
+
+#### Classifier
+- **SafeNet Trusted Access – Account Lockout Alert Classifier**
+- **SafeNet Trusted Access – Account Lockout Alert Mapper**
+
+#### Incident Fields
+- **SafeNet Trusted Access - Instance Name**
+- **SafeNet Trusted Access - Username**
+
+#### Incident Types
+- **SafeNet Trusted Access Alert**
+- **SafeNet Trusted Access Account Lockout Alert**
+
+#### Playbooks
+##### New: SafeNet Trusted Access - Terminate User SSO Sessions
+- This playbook terminates user's SSO sessions so that upon the next login attempt following the unlocking of the account, authentication is required. (Available from Cortex XSOAR 6.0.0).
+
+##### Updated: SafeNet Trusted Access - Add to Unusual Activity Group
+- This playbook adds the user to a group that was created to identify unusual activity. SafeNet Trusted Access policies can be configured to take this into account and provide stronger protection when handling access events from users who are members of the group. The user is added to this group for a configurable period of time.
```

Comment:
```suggestion
- ***Breaking Change:*** Changed playbook inputs names. 
```

Reply:
Done. As discussed, updated release note and added the **.json** file.